src/super_gradients/training/models/detection_models/customizable_detector.py

```diff
@@ -128,7 +128,7 @@ def set_dataset_processing_params(
         self._default_nms_iou = iou or self._default_iou
         self._default_nms_conf = conf or self._default_conf
 
-    def predict(self, images, iou: Optional[List[str]] = None, conf: Optional[List[str]] = None) -> DetectionResults:
+    def predict(self, images, iou: Optional[float] = None, conf: Optional[float] = None) -> DetectionResults:
```

Comment:
`images` argument is not annotated

Reply:
Added in 38349a7361938fbf7e5ce5c490c84f0eaf542afd